package.json

```diff
@@ -180,7 +180,7 @@
 		"webpack-filter-warnings-plugin": "^1.2.1",
 		"webpack-hot-middleware": "^2.25.0",
 		"webpack-node-externals": "^1.7.2",
-		"webpack-svgstore-plugin": "^4.1.0"
+		"webpack-svgstore-plugin": "ryan-ludwig/webpack-svgstore-plugin#develop"
```

Comment:
This is unfortunate, but the only way I could get the sprite plugin to work with our magnifying glass icon was to fork it and update it's SVGO dependency. I have a PR up with them. 

I believe @BoulderBrains has had problems adding SVGs to the sprite in the past, I believe this is the same issue.

Reply:
Can you link to that PR?

Reply:
https://github.com/mrsum/webpack-svgstore-plugin/pull/180

Reply:
I think webpack-svgstore-plugin is practically deprecated. We should try to replace it with something else

Reply:
Yeah, I've wondered if we shouldn't just do the sprite manually since it has so few items now.

Reply:
I have my doubts that webpack-svgstore-plugin is going to merge my PR. Don't really love that this is pointing to my personal github for something we're wanting to get into prod.

2 options I think:
1) Fork webpack-svgstore-plugin to the kiva github, make the same change, point package.json to that.  
2) Ditch webpack-svgstore-plugin, roll the sprite by hand (easy), and ajax it in from TheHeader.vue or something similar.

Both have upsides and downsides. @emuvente @mcstover thoughts?

Reply:
I'm ok with #1.

Alternatively, if we take the #2 approach could we just import and render the image in App.vue to ensure it's always present?

Reply:
Going to run with #1 for today to get this big PR into dev.

With #2, yeah I think putting the sprite directly into App.vue makes sense to me 👍.